src/components/executive-education-2u/components/CourseSummaryCard.jsx

```diff
@@ -10,9 +10,7 @@ import { numberWithPrecision } from '../../course/data/utils';
 import { DATE_FORMAT, ZERO_PRICE } from '../../course/data/constants';
 import { useMinimalCourseMetadata } from '../../course/data/hooks';
 
-const CourseSummaryCard = ({ enrollmentCompleted }) => {
-  const { data: minimalCourseMetadata } = useMinimalCourseMetadata();
-
+const CourseSummaryCard = ({ minimalCourseMetadata, enrollmentCompleted }) => {
```

Comment:
Nit: Similarly here, any particular reason to not call useMinimalCourseData directly within this component versus passing it as a prop?